### PR TITLE
add vip to webhook e2e

### PR DESCRIPTION
**Pull request by ShaPoHun**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
Examples of user facing changes:
- Features
add vip to webhook e2e

### CODE REVIEW

File: test/e2e/framework/vip.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	ExpectNoError(err, "Error creating vip")
+	return vip.DeepCopy()
+}
+
+// Patch patches the vip
+func (c *VipClient) Patch(original, modified *apiv1.Vip, timeout time.Duration) *apiv1.Vip {
+	patch, err := util.GenerateMergePatchPayload(original, modified)
+	ExpectNoError(err)
+
+	var patchedVlan *apiv1.Vip
```
Comment (hongzhen-ma) [inline test/e2e/framework/vip.go:46]: patchedVlan ？

File: test/e2e/framework/vip.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	return &VipClient{
+		f:            f,
+		VipInterface: f.KubeOVNClientSet.KubeovnV1().Vips(),
+	}
+}
+
+func (c *VipClient) Get(name string) *apiv1.Vip {
+	vip, err := c.VipInterface.Get(context.TODO(), name, metav1.GetOptions{})
+	ExpectNoError(err)
+	return vip
```
Comment (zhangzujian) [inline test/e2e/framework/vip.go:31]:
```suggestion
	return vip.DeepCopy()
```

**Comment by zhangzujian** [comment]
From which version VIP is introduced? We should skip the test if the feature is not supported.

**Comment by ShaPoHun** [comment]
> From which version VIP is introduced? We should skip the test if the feature is not supported.

Support for vip Webhooks starts with tag v1.11.2

**Comment by zhangzujian** [comment]
Since webhook e2e is running only on the master branch, there is no need to check the version.

**Review by hongzhen-ma — APPROVED** [review]

**Pull request merged**